docs/geedocs/answer/4485225.html

```diff
@@ -268,32 +239,20 @@ <h3 id="google_earth_ec">Google Earth EC</h3>
   <p>Launch the Google Earth EC client. When prompted for a server address, enter <code>http://localhost:9335</code>. If you've changed the default port in portable.cfg, use the new port value instead.</p>
   <hr/>
   <h3 id="google_earth_api">Google Earth API</h3>
-  <p>The Google Earth Enterprise Portable Server comes with a preconfigured HTML page that displays your globe using the Earth API. This page is at <code>http://localhost:9335/earth/earth_local.html</code>.</p>
+  <p>The Google Earth Enterprise Portable Server comes with a preconfigured HTML page that displays your globe using the Earth API. The page relies on the discontinued Google Earth browser plug-in to render globes, so it doesn't work in most recent browsers. This page is at <code>http://localhost:9335/earth/earth_local.html</code>.</p>
   <p>If you'd like to make your own custom Earth API application, make a copy of the included file from which to start.</p>
   <p>You can also reach the page listed above by clicking <strong>view in browser</strong> from the administration page at <code>http://localhost:9335</code>.</p>
   <hr/>
+  <!--
```

Comment:
Oh, I can delete that. I wasn't sure if the new Portable Server no longer supports key authentication. It seems that it doesn't, but I haven't looked through the code thoroughly enough to be sure.